hosts.bat

```diff
@@ -0,0 +1,5 @@
+echo gradlew.bat environmentHosts -q > .gap.hosts.tmp
+
+powershell -command "Start-Process cmd -ArgumentList '/C cd %CD% && type .gap.hosts.windows >> C:\Windows\System32\drivers\etc\hosts' -Verb runas"
```

Comment:
There is no way to set var then echo and redirect it to file as in Unix?

Reply:
There is, however, using tmp file is much simpler and cleaner